src/geodesic.c

```diff
@@ -85,7 +85,10 @@ static void Init() {
     tolb = tol0 * tol2;
     xthresh = 1000 * tol2;
     degree = pi/180;
-    NaN = sqrt(-1.0);
+    {
```

Comment:
Thanks, I'll backport these changes.  Also I note that Visual Studio gives two warnings
that S12 "may" be uninitialized.  It's wrong! but I initialize it to 0 in those places together
with a comment /* Initialize S12 to stop Visual Studio warning */